### PR TITLE
Task-51602: Add screens for documents empty for both desktop and mobi…

### PR DESCRIPTION
**Pull request by lbengayed**
…le view

### CODE REVIEW

File: documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue [hunk 1]
```diff
@@ -5,20 +5,26 @@
...
-        :page-size="pageSize"
-        :offset="offset"
-        :limit="limit"
-        :has-more="hasMore"
-        :sort-field="sortField"
-        :ascending="ascending"
-        :loading="loading" />
+      <div v-if="!filesLoad">
+        <documents-header-left
+          class="py-2" />
```
Comment (mdiha) [inline documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue:10]: Why the isMobile prop is removed ?

File: documents-webapp/src/main/webapp/vue-app/documents/components/header/actions/DocumentsAddNewFile.vue [hunk 2]
```diff
@@ -8,7 +8,7 @@
 export default {
   computed: {
     isMobile() {
-      return this.$vuetify.breakpoint.name === 'xs';
+      return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
```
Comment (mdiha) [inline documents-webapp/src/main/webapp/vue-app/documents/components/header/actions/DocumentsAddNewFile.vue:11]: Isn't 'sm' breakpoint for tablets ? do we have the same design for tablets and mobile

File: documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue [hunk 3]
```diff
@@ -5,20 +5,26 @@
...
+        <documents-no-body />
+      </div>
+      <div v-else>
+        <documents-header
+          class="py-2" />
```
Comment (mdiha) [inline documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue]: The isMobile prop has to be passed to the 'documents-header' component it is needed.

File: documents-webapp/src/main/webapp/vue-app/documents/components/header/actions/DocumentsAddNewFile.vue (hunk 2 above)
Comment (lbengayed) [reply documents-webapp/src/main/webapp/vue-app/documents/components/header/actions/DocumentsAddNewFile.vue:11]: Yes its the same for mobile and tablet

File: documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue (hunk 1 above)
Comment (lbengayed) [reply documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue:10]: it is a mistake when i rebase with branch feature/document, i will add it

**Review by mdiha — APPROVED** [review]
LGTM

**Pull request merged**